src/styles/blocks/header.scss

```diff
@@ -38,5 +40,11 @@
     text-transform: uppercase;
     font-size: 12px;
     font-weight: 500;
+
+    transition: color 0.3s;
```

Comment:
create a variable for transition-duration in case you've already used this value a few times